src/main/scala/com/allantl/jira4s/v2/domain/WorkLogPayLoad.scala

```diff
@@ -0,0 +1,67 @@
+package com.allantl.jira4s.v2.domain
+
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+
+case class WorkLogPayLoad(comment: Option[String], started: String, timeSpentSeconds: Int)
+
+object WorkLogPayLoad {
+  implicit val issueDecoder: Decoder[WorkLogPayLoad] = deriveDecoder
+  implicit val issueEncoder: Encoder[WorkLogPayLoad] = deriveEncoder
+}
+
+case class WorkLogCreateResponse(id: String, self: String)
+
+object WorkLogCreateResponse {
+  implicit val issueDecoder: Decoder[WorkLogCreateResponse] = deriveDecoder
+  implicit val issueEncoder: Encoder[WorkLogCreateResponse] = deriveEncoder
+}
+
+sealed trait AdjustEstimate
```

Comment:
This should be in a different file. Under /domain/enums. 
Also ADT should be inside an object.
```
sealed trait AdjustEstimate

object AdjustEstimate {
    case class NewEstimate(newEstimate: String) extends AdjustEstimate
}

```